operator/v1/types_clustermanager.go

```diff
@@ -118,6 +118,26 @@ type RegistrationHubConfiguration struct {
 	RegistrationDrivers []RegistrationDriverHub `json:"registrationDrivers,omitempty"`
 }
 
+const (
+	// AwsIrsaAuthType represents the authentication type that uses AWS IRSA
+	AwsIrsaAuthType = "awsirsa"
+	// CSRAuthType represents the authentication type that uses Kubernetes CSR
+	CSRAuthType = "csr"
+	// GRPCAuthType represents the authentication type that uses gRPC.
+	GRPCAuthType = "grpc"
+)
+
+// GRPCAuthSigner is the signer name used when creating CSRs for gRPC authentication.
+const GRPCAuthSigner = "open-cluster-management.io/grpc"
+
+const (
+	// CSRUsernameAnnotation is added to a CSR to identify the user who requested the CSR.
+	CSRUsernameAnnotation = "open-cluster-management.io/csr/user"
+
+	// CSRAddOnAnnotation is added to a CSR to identify the name of the addon  that requested the CSR.
+	CSRAddOnNameAnnotation = "open-cluster-management.io/csr/addon"
```

Comment:
I think this has been added on the label.

Reply:
we have this on the label, so we can 
- check whether the username is from the gRPC server.
- check that the `addon.Name` matches the CSR label.